marklogic-data-hub/src/main/java/com/marklogic/hub/step/RunStepResponse.java

```diff
@@ -198,7 +198,7 @@ public void setStepEndTime(String stepEndTime) {
     public String toString() {
         return String.format("[flowName: %s, stepName: %s, stepDefinitionName %s, stepDefinitionType %s, success: %s, " +
                 "status: %s, totalEvents: %d, successfulEvents: %d, " + "failedEvents: %d, successfulBatches: %d, " +
-                "failedBatches: %d, stepStartTime: %d , stepEndTime: %d]", flowName, stepName, stepDefinitionName,
+                "failedBatches: %d, stepStartTime: %s , stepEndTime: %s]", flowName, stepName, stepDefinitionName,
             stepDefinitionType, String.valueOf(success), status, totalEvents, successfulEvents, failedEvents,
             successfulBatches, failedBatches, stepStartTime, stepStartTime);
```

Comment:
The last line has to be **successfulBatches, failedBatches, stepStartTime, stepEndTime);**